credsweeper/__main__.py

```diff
@@ -270,20 +271,30 @@ def get_json_filenames(json_filename: str):
     return added_json_filename, deleted_json_filename
 
 
-def scan(args: Namespace, content_provider: AbstractProvider, json_filename: Optional[str],
-         xlsx_filename: Optional[str]) -> int:
+def scan(args: Namespace,
+         content_provider: AbstractProvider,
+         json_filename: Optional[str] = None,
+         xlsx_filename: Optional[str] = None) -> int:
     """Scan content_provider data, print results or save them to json_filename is not None
 
     Args:
         args: arguments of the application
         content_provider: FilesProvider instance to scan data from
-        json_filename: json type report file path or None
-        xlsx_filename: xlsx type report file path or None
+        json_filename: (Deprecated) json type report file path or None
+        xlsx_filename: (Deprecated) xlsx type report file path or None
```

Comment:
It seems `json_filename` and `xlsx_filename` can be removed from `scan()`..
Could you please consider to remove those two parameters?